_posts/2025-09-13-22.0.0-release.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: "Apache Arrow 22.0.0 Release"
+date: "2025-09-13 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 22.0.0 release. This release
+covers over 3 months of development work and includes [**XXX resolved
+issues**][1] on [**YYY distinct commits**][2] from [**ZZ distinct
+contributors**][2]. See the [Install Page](https://arrow.apache.org/install/) to
+learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 21.0.0 release, Kyle Barron has been invited to be committer.
+
+Matthijs Brobbel, Adam Reeve and Rossi Sun have been joined the
+Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+The first Apache Arrow Summit was held on October 2nd 2025 in Paris, France
+as part of [PyData Paris](https://pydata.org/paris2025).
+Program details and agenda are here: https://www.meetup.com/pydata-paris/events/310646396/ 
+
+There were around 35 attendees, of which ~20 were existing core developers or PMC members.
+The Summit was overwhelmingly described as a success, with a friendly atmosphere between all participants.
+Unfortunately, no A/V recording system was available for this event.
+
+## Arrow Flight RPC Notes
+
+Support for dictionary replacement and dictionary encoding has been added to the DoGet and DoExchange methods. ([GH-45056](https://github.com/apache/arrow/issues/45056),[GH-45055](https://github.com/apache/arrow/issues/45055) and [GH-26727](https://github.com/apache/arrow/issues/26727)).
+
+As part of supporting dictionary replacement we have also exposed the `ipc::ReadStats` on the `FlightStreamReader` in order to facilitate debugging. ([GH-47422](https://github.com/apache/arrow/issues/47422))
+
+An initial ODBC framework implementation has been merged. ([GH-47516](https://github.com/apache/arrow/issues/47516))
```

Comment:
We may not need this because ODBC support isn't usable yet.

Reply:
That makes sense, I've removed the ODBC note